src/params.rs

```diff
@@ -273,7 +270,7 @@ impl<PRFH: Hasher + Clone, MSGH: Hasher + Clone> Params<PRFH, MSGH> {
 
             let mut hasher = PRFH::new();
             hasher.write(p_seed.to_vec());
-            hasher.write(vec![j as u8]);
+            hasher.write(vec![(j + 1) as u8]);
```

Comment:
Is this `j + 1` always `< 255`?

Reply:
yes, the highest it can go is `(W-1)` which is 255